doc/autodocs/doxygen.conf

```diff
@@ -17,7 +17,7 @@ INPUT = ../../storage ../../storage/Devices ../../storage/Holders		\
 FILE_PATTERNS = Actiongraph.h BlkDevice.h Device.h Devicegraph.h Disk.h		\
 	Encryption.h Environment.h Ext4.h Filesystem.h Gpt.h Holder.h		\
 	HumanString.h Msdos.h Partition.h PartitionTable.h Region.h Storage.h	\
-	StorageInterface.h Subdevice.h Swap.h User.h Logger.h LvmVg.h LvmLv.h
+	Subdevice.h Swap.h User.h Logger.h LvmVg.h LvmLv.h
```

Comment:
I see. Depending on the future of the legacy interface we can split the enums into a separate file.